server/api.go

```diff
@@ -60,13 +60,13 @@ func (p *Plugin) getGo(w http.ResponseWriter, r *http.Request) {
 	}
 
 	source := r.URL.Query().Get("source")
-	if len(source) < 2 || len(source) > 4 {
+	if len(source) < 2 || len(source) > 5 {
```

Comment:
Should we just check for 
```suggestion
	if len(source) != 2 && len(source) != 5 {
```

here and below to be more strict?